### PR TITLE
[OMP] #6609 Fix multiple submissions not taken into account when ONIX plugin tried to export.

### PR DESCRIPTION
**Pull request by defstat**

### CODE REVIEW

File: plugins/importexport/onix30/filter/MonographONIX30XmlFilter.inc.php [hunk 1]
```diff
@@ -75,16 +72,24 @@ function &process(&$monograph) {
 		return $doc;
 	}
 
+	function createSubmissionNode($doc, $rootNode, $monograph) {
```
Comment (asmecher) [inline plugins/importexport/onix30/filter/MonographONIX30XmlFilter.inc.php:75]: Needs code self-documentation

File: plugins/importexport/onix30/filter/filterConfig.xml [hunk 2]
```diff
@@ -16,7 +16,7 @@
 			symbolic="monograph=>onix30-xml"
 			displayName="plugins.importexport.onix30.displayName"
 			description="plugins.importexport.onix30.description"
-			inputType="class::classes.submission.Submission"
+			inputType="class::classes.submission.Submission[]"
```
Comment (asmecher) [inline plugins/importexport/onix30/filter/filterConfig.xml:19]: I believe you'll have to add a migration for this -- see e.g.: https://github.com/pkp/omp/commit/ec086ceeb36e6f94de5a90375748df29be7b62a0#diff-1eb4b692ad50b43e53d5133069db2bd2f7cfe10950de0f6e8972f071505860e8R49

**Review by asmecher — CHANGES_REQUESTED** [review]
Just a couple of small changes, then it's ready for merge!

File: plugins/importexport/onix30/filter/MonographONIX30XmlFilter.inc.php (hunk 1 above)
Comment (defstat) [reply plugins/importexport/onix30/filter/MonographONIX30XmlFilter.inc.php]: Fixed.

File: plugins/importexport/onix30/filter/filterConfig.xml (hunk 2 above)
Comment (defstat) [reply plugins/importexport/onix30/filter/filterConfig.xml:19]: I knew I was missing something there. Fixed.

Do you think that we could consider a more "automatic" way to do that, from the `upgrade.php` script, without the need of the migration?

Comment (asmecher) [reply plugins/importexport/onix30/filter/filterConfig.xml:19]: Yes, but I'd like to revisit the filter framework more generally before we do that. In my opinion there's not much gained by having a persistence layer involved, and it introduces a lot of complexity and confusion.

**Pull request merged**